src/main/resources/assets/iris/lang/ru_ru.json

```
@@ -26,10 +28,22 @@
   "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" уже находится в вашей папке с шейдерами!",
   "options.iris.shaders.disabled": "Шейдеры: Выключены",
   "options.iris.shaders.enabled": "Шейдеры: Включены",
+  "options.iris.back": "Назад",
+  "options.iris.reset": "Сбросить",
+  "options.iris.reset.tooltip": "Сбросить ВСЕ параметры и применить?",
+  "options.iris.reset.tooltip.holdShift": "Удерживайте SHIFT и нажмите, чтобы сбросить",
+  "options.iris.setToDefault": "Установить параметр по умолчанию?",
+  "options.iris.profile": "Профиль",
+  "options.iris.profile.custom": "Пользовательский",
   "options.iris.shadowDistance": "Максимальное расстояние для теней",
   "options.iris.shadowDistance.enabled": "Позволяет вам изменить максимальное расстояние для теней. Блоки и существа за этим расстоянием не будут иметь теней. Снижая этот параметр, вы можете кардинально улучшить производительность.",
   "options.iris.shadowDistance.disabled": "Ваши текущие шейдеры уже установили своё расстояние для теней; вы не сможете её изменить.",
+  "options.iris.shadowDistance.sodium_tooltip": "Расстояние видимости теней определяет, насколько далеко местность может обрабатываться на этапе просчёта теней. Меньшие значения соответствуют меньшей площади, что улучшает частоту кадров. Этот параметр нельзя изменить для пакетов, в которых установлено своё расстояние рендеринга теней. Фактическое расстояние рендеринга ограничено дальностью прорисовки.",
```

Вместе с изменением текста в названии настройки нужно изменить текст в соответствующей ей подсказке. + предлагаю заменить "пакетов" на "шейдеров", звучит  лучше, опять же.

```suggestion
  "options.iris.shadowDistance.sodium_tooltip": "Дальность теней определяет, насколько далеко местность может обрабатываться на этапе просчёта теней. Меньшие значения соответствуют меньшей площади, что улучшает частоту кадров. Этот параметр нельзя изменить для шейдеров, в которых установлено своё расстояние рендеринга теней. Фактическое расстояние рендеринга ограничено дальностью прорисовки.",
```

---

Ок, спасибо, всё исправлю